openedx/core/djangoapps/credentials/management/commands/tests/test_notify_credentials.py

```diff
@@ -286,7 +287,10 @@ def test_args_from_database(self, mock_task):
 
         # Told to use it, and enabled. Should use config in preference of command line
         self.expected_options['start_date'] = '2017-03-01T00:00:00Z'
-        del self.expected_options['skip_checks']
+        if django.VERSION >= (3, 0):
+            self.expected_options['skip_checks'] = False
```

Comment:
A new flag is added in Django management commands so when we don't use that it will still be present in options with a false value 

> The new --skip-checks option skips running system checks prior to running the command.

https://docs.djangoproject.com/en/3.2/releases/3.0/#management-commands